### PR TITLE
Rename bar frames (lowercase, s -> is)

### PR DESCRIPTION
**Pull request by iibrahimli**

### CODE REVIEW

File: marimbabot_ur5_moveit_config/config/arm_controller_ur5.yaml [hunk 1]
```diff
@@ -17,12 +17,12 @@ pos_joint_traj_controller:
   constraints:
     goal_time: 0.6
     stopped_velocity_tolerance: 0.05
-    ur5_shoulder_pan_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_shoulder_lift_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_elbow_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_wrist_1_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_wrist_2_joint: {trajectory: 0.1, goal: 0.01}
-    ur5_wrist_3_joint: {trajectory: 0.1, goal: 0.01}
+    ur5_shoulder_pan_joint: {trajectory: 0.1, goal: 0.1}
```
Comment (ateRstones) [inline marimbabot_ur5_moveit_config/config/arm_controller_ur5.yaml:20]: Why did you change these values?

**Review by ateRstones — COMMENTED** [review]
Looks good to me other than the comment I added

File: marimbabot_ur5_moveit_config/config/arm_controller_ur5.yaml (hunk 1 above)
Comment (iibrahimli) [reply marimbabot_ur5_moveit_config/config/arm_controller_ur5.yaml:20]: Current PID values make it impossible to follow the trajectory. This is just for simulation to work at the moment, will be changed.

**Review by scaredycode — APPROVED** [review]
Looks good, I will change the note names in the behavior/interpreter accordingly 👍

**Comment by scaredycode** [comment]
I pushed my commit to your branch because it is a very small fix and closely related to your PR, hope you don't mind 😊

**Comment by iibrahimli** [comment]
@scaredycode Thanks for the fix!

**Comment by Flova** [comment]
Please also change the definition comment in the message definition file for the hit sequence element.
